k8s/crds.yaml

```diff
@@ -788,12 +795,18 @@ spec:
                 description: The error messages when calling longhorn engine on listing or inspecting backups.
                 nullable: true
                 type: object
+              newlyUploadDataSize:
+                description: Size of newly uploaded data
+                type: string
               ownerID:
                 description: The node ID on which the controller is responsible to reconcile this backup CR.
                 type: string
               progress:
                 description: The snapshot backup progress.
                 type: integer
+              reUploadedDataSize:
+                description: Size of reuploaded data
+                type: string
```

Comment:
Just found the crd.yaml is not regenerated. Also need to update to https://github.com/longhorn/longhorn/pull/8290/files

Reply:
fixed.